README.md

```diff
@@ -933,6 +933,92 @@ _syscall.number()_ returns the system call number referenced by the given `name`
 
 The `syscall.table` library provides support for translating system call names to addresses (light userdata).
 
+### netfilter
+
+The `netfilter` library provides support for the kernel [Netfilter](https://netfilter.org/) subsystem. This includes support for xtables (`match`, `target`) and netfilter hooks.
+
+#### `netfilter.xtable(name)`
+
+_netfilter.xtable(name)_ creates a new `netfilter.xtable` object for the specified `name`. The 'name' must be unique and must not be used by any other xtable. This name will be used to create userspace and kernel objects for the xtable when `match` and `target` objects are added to the xtable.
+
+#### `netfilter.match(match_func), nf:match(match_func)`
+
+_netfilter.match(match_func)_ registers the specified `match_func`. The `match_func` is a function that will be called when a packet is processed by the xtable. The function must return boolean value indicating whether the packet should be accepted or dropped.
```

Comment:
it actually should return any netfilter action (e.g., `NF_STOLEN`) instead only drop/accept, right?

Reply:
`xt_register_match` takes `struct xt_match` as an argument :

```c
          /* Return true or false: return FALSE and set *hotdrop = 1 to
             force immediate packet drop. */
          /* Arguments changed since 2.6.9, as this must now handle
             non-linear skb, using skb_header_pointer and
             skb_ip_make_writable. */
          bool (*match)(const struct sk_buff *skb,
                        struct xt_action_param *); 
```

This is the callback signature from `struct xt_match`